package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "chef-vsts-tasks",
+  "name": "habitat-tasks",
   "description": "Chef Integration for Visual Studio Team System (VSTS)",
```

Comment:
Update to Azure Pipelines

Reply:
Damn missed one :-)

Reply:
Sorry I didn't
I have taken the name of the buikd platform out so we are not dependent on it
Have a look at the docs for the piple and you will see why